xml_inscriptions/metadata_only/MO.Col.UM.MAA.L.59.2.xml

```diff
@@ -75,7 +75,7 @@
                <bibl>
                   <ref type="external"
                      target="http://maacollections.missouri.edu/ArgusNET/Portal.aspx?lang=en-US&amp;g_AAEM=Arretine+Sigillata+Plate+59.2"
-                     >Museum of Art and Archeology (Web Entry)</ref>
+                     >Museum of Art and Archeology</ref>
```

Comment:
See query above; if this is going to be a change that we're making across all inscriptions, tell me and I'll make a global change